### DISCUSSION TITLE
Could this tool work on my other ‘’special monitor‘’？

### DISCUSSION THREAD

**Discussion by MiuKuou** [Tyrrrz/LightBulb#314] [Q&A]
I have two monitors, but the extra screen is linked to my computer not through a typical method, but tow apps, Sunshine and Moonlight.
during the use of the application, I found there's an influence on my mian computer screen, but for the other, it just doesn't work. 

(https://github.com/MiuKuou/picture/blob/master/tmpDBC2.png)

**Comment by Tyrrrz** [comment]
I doubt it, I imagine both monitors need to be connected to the same graphics adapter
